lib/screens/container/driver_list.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:shipanther/bloc/container/container_bloc.dart';
+import 'package:shipanther/helper/text_widget.dart';
+import 'package:shipanther/l10n/shipanther_localization.dart';
+
+import 'package:shipanther/widgets/shipanther_scaffold.dart';
+import 'package:trober_sdk/api.dart' as api;
+
+class DriverContainerList extends StatelessWidget {
+  final ContainerBloc containerBloc;
+  final ContainersLoaded containerLoadedState;
+  final api.User loggedInUser;
+  const DriverContainerList(
+    this.loggedInUser, {
+    Key key,
+    @required this.containerLoadedState,
+    this.containerBloc,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    var title = ShipantherLocalizations.of(context).containersTitle;
+    List<Widget> actions = [];
+    Widget body = ListView.builder(
+      itemCount: containerLoadedState.containers.length,
+      itemBuilder: (BuildContext context, int index) {
+        var t = containerLoadedState.containers.elementAt(index);
+
+        return Column(
+          children: [
+            ExpansionTile(
+              leading: Icon(Icons.home_work),
+              trailing: Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  Text(
+                    DateFormat('dd-MM-yyyy').format(t.reservationTime),
+                    style: TextStyle(
+                        fontSize: 15, color: Color.fromRGBO(204, 255, 0, 1)),
+                  ),
+                  Text(
+                    DateFormat('kk:mm').format(t.reservationTime),
+                    style: TextStyle(fontSize: 15),
+                  ),
+                ],
+              ),
+              title: Text(
+                t.serialNumber,
+                style: TextStyle(
+                    color: t.status == api.ContainerStatus.accepted
+                        ? Colors.red
+                        : Colors.white),
+              ),
+              subtitle: Text('${t.origin} to ${t.destination}'),
+              children: [
+                Column(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    textSpan('Assigned by: ', t.tenantId, c2: Colors.orange),
+                    textSpan('Size: ', t.size.toString(), c2: Colors.deepPurple)
```

Comment:
The colors make it very hard to read. Can we stick with default colors until the color has some significance.